intl/msg_hash_us.h

```diff
@@ -6674,6 +6674,16 @@ MSG_HASH(
 
 /* Quick Menu > Shaders > Save */
 
+MSG_HASH(
+   MENU_ENUM_LABEL_VALUE_VIDEO_SHADER_PRESET_SAVE_REFERENCE,
+   "Save Simple Preset"
```

Comment:
I think we need to change the label here. Every existing entry in this menu starts with `Save `, and each of them triggers a save action - this is an option rather than an action, so it should be differentiated visually. Perhaps it could just be `Simple Presets`...?